tests/behat/Paragraphs/WYSIWYG.feature

```diff
@@ -0,0 +1,159 @@
+    And I should not see a "footer" element in the "content" region
+    And I should see 2 "code" elements in the "content" region
+    And I should see 1 "dl" element in the "content" region
+    And I should see 2 "dt" element in the "content" region
+    And I should see 2 "dd" element in the "content" region
+
+    # List Tags
+    And I should see an "ul li:contains('This is a list')" element
+    And I should see an "ul ul li:contains('child list items')" element
+    And I should see an "ol li:contains('Ordered list item')" element
+    And I should see an "ol ol li:contains('Child ordered list item')" element
+
+    # Table Tags
+    And I should see 1 "table" element in the "content" region
+    And I should see 1 "caption" element in the "content" region
+    And I should see 1 "tbody" element in the "content" region
+    And I should see 3 "tr" elements in the "content" region
+    And I should see 2 "th[scope]" elements in the "content" region
+    And I should see 4 "td" element in the "content" region
+
+  @api @javascript
+  Scenario: Test Embedded Image
+    Given I am logged in as a user with the "administrator" role
+    And I am on "/node/add/stanford_page"
+    Then I fill in "Title" with "Test WYSIWYG Paragraph"
+    And I press "Add WYSIWYG"
+    And I wait for AJAX to finish
+    Then I click the "a[title='Embed Media']" element
+    And I wait for AJAX to finish
+    Then I switch to "entity_browser_iframe_media_browser" iframe
+    Then I click "Embed a File"
+    And I wait for AJAX to finish
+    And I wait 1 seconds
+    Then I drop "../assets/images/logo.jpg" file into dropzone
+    And I press "Add to Library"
+    And I wait for AJAX to finish
+    And I wait 1 seconds
+    Then I press "Continue"
+    And I wait for AJAX to finish
+    And I wait 1 seconds
+    Then I exit iframe
+    And I wait 2 seconds
+    Then I select "Medium (220×220)" from "Image Style"
+    And I fill in "Alternate text" with "Stanford Logo"
+    Then I click the ".entity-select-dialog .form-actions button" element
+    And I wait for AJAX to finish
+    Then I press "Save"
+    Then I should be on "/test-wysiwyg-paragraph"
+    And I should see 1 "img" element in the "content" region
+    And the element "img" should have the attribute "alt" with the value "Stanford Logo"
+
+  @api @javascript @testthis
+  Scenario: Test Embedded Video
+    Given I am logged in as a user with the "administrator" role
+    And I am on "/node/add/stanford_page"
+    Then I fill in "Title" with "Test WYSIWYG Paragraph"
+    And I press "Add WYSIWYG"
+    And I wait for AJAX to finish
+    Then I click the "a[title='Embed Media']" element
+    And I wait for AJAX to finish
+    Then I switch to "entity_browser_iframe_media_browser" iframe
+    Then I click "Embed External Content"
+    And I wait for AJAX to finish
+    And I wait 1 seconds
+    Then I fill in "Shareable Url" with "https://www.youtube.com/watch?v=ktCgVopf7D0"
```

Comment:
This item stays in the media library after the tests.

Reply:
It gets removed on my test runs.

Reply:
I'm not worried about artifacts. Tests are not to be run on production sites.